libs/feature/dataviz/src/lib/chart-view/chart-view.component.ts

```diff
@@ -74,6 +84,7 @@ export class ChartViewComponent {
         }),
         finalize(() => {
           this.loading = false
+          this.changeDetector.detectChanges()
```

Comment:
it's tricky, for me it has to do with the fact that in the case of the table in case of an error we still emit an empty array; here we use `EMPTY` which means that nothing will trigger the change detection unless we do it manually